src/domain/community/invitation.external/invitation.external.entity.ts

```diff
@@ -0,0 +1,34 @@
+import { Column, Entity, ManyToOne } from 'typeorm';
+import { Community } from '@domain/community/community/community.entity';
+import { IInvitationExternal } from './invitation.external.interface';
+import { AuthorizableEntity } from '@domain/common/entity/authorizable-entity';
+@Entity()
+export class InvitationExternal
+  extends AuthorizableEntity
+  implements IInvitationExternal
+{
+  @ManyToOne(() => Community, community => community.externalInvitations, {
+    eager: false,
+    cascade: false,
+    onDelete: 'CASCADE',
+  })
+  community!: Community;
+
+  @Column('varchar', { length: 255, nullable: false })
+  email = '';
+
+  @Column('varchar', { length: 255, nullable: true })
+  firstName = '';
+
+  @Column('varchar', { length: 255, nullable: true })
+  lastName = '';
```

Comment:
Is there a reason to have these this long?

Reply:
Basically copying over what is there on the user entity, so that if we later wanted to re-use this for actually creating the user profile we could...